config/owasp/suppressions.xml

```diff
@@ -36,5 +36,6 @@
   <suppress until="2023-05-26">
     <cve>CVE-2023-28708</cve>
     <cve>CVE-2023-20861</cve>
+    <cve>CVE-2023-20860</cve>
```

Comment:
Make sure you also open a ticket and a PR separately to address this one properly. PET might want to look into that ticket so once you created it, drop a line to Sabah